docs/build/references/platform-transaction-serialization.md

```diff
@@ -1819,4 +1819,159 @@ Let’s make a subnet auth:
   // address index 1
   0x00, 0x00, 0x00, 0x00
 ]
+```
```

Comment:
This location of this doc has been changed, please merge master branch and make proper changes.